README.md

```diff
@@ -188,6 +188,7 @@ This endpoint is used to send a batch of messages. It accepts a JSON payload con
   {
     "data": {
       "type": "MessageBatch",
+      "messageOriginKey": "some-origin-team",
```

Comment:
I know this PR is still in draft but before we go further with it I think we should be identifying the calling service by a header value and not passing the key in the post body. This is fairly standard practice when passing things like bearer tokens and HMAC signatures when consuming APIs. It also keeps the post body identical to that of NHS Notify meaning we can validate the request using their schema.